### PR TITLE
Clean up Jenkinsfile

### PR DESCRIPTION
**Pull request by aelred**
This one will make more sense commit-by-commit

### CODE REVIEW

**Comment by aelred** [comment]
!ci - not sure I understand what happened

**Comment by aelred** [comment]
@fppt all ready to review! This one might need to be read commit-by-commit to understand...

File: Jenkinsfile [hunk 1]
```diff
@@ -52,12 +41,12 @@ def slackGithub(String message, String color = null) {
     slackSend channel: "#github", color: color, message: formattedMessage
 }
 
-def runIntegrationTest(String workspace, String moduleName) {
-    String modulePath = "${workspace}/grakn-test/${moduleName}"
+def runIntegrationTest(String moduleName) {
+    String modulePath = "${pwd()}/grakn-test/${moduleName}"
```
Comment (fppt) [inline Jenkinsfile]: Kind of find `workspace` easier to work with. I hate "embedded paths" but that's just me.

File: Jenkinsfile [hunk 2]
```diff
@@ -133,28 +115,24 @@ def shouldDeployLongRunningInstance() {
 }
 
 def mvn(String args) {
-    sh "mvn ${args}"
+    sh "mvn --batch-mode ${args}"
```
Comment (fppt) [inline Jenkinsfile]: Why the addition of `--batch-mode`?

File: Jenkinsfile [hunk 3]
```diff
@@ -266,3 +252,12 @@ if (shouldRunAllTests()) {
         slackGithub "Build Success", "good"
     }
 }
+
+try {
+    runBuild()
+} catch (Exception e) {
+    node {
+        slackGithub "Build Failure", "danger"
```
Comment (fppt) [inline Jenkinsfile]: Fair point. Danger is afoot.

File: Jenkinsfile (hunk 2 above)
Comment (aelred) [reply Jenkinsfile]: It tells maven that we're not in an interactive shell, so it shouldn't ever prompt, or print fancy download bars that refresh or anything.

File: Jenkinsfile (hunk 1 above)
Comment (aelred) [reply Jenkinsfile]: Yeah, I'm not sure which is better. I'll try changing it back.

**Comment by aelred** [comment]
@fppt all ready 👍

**Pull request merged**